docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/entrypoints/ncids-application-page/ncids-application-page.scss

```diff
@@ -0,0 +1,101 @@
+// file should generate markup.
+@forward '../cgdp-required.scss';
+@forward '../../lib/components/usa-breadcrumb';
+@forward '../../lib/components/usa-sidenav';
```

Comment:
So if you are going to implement both with a left nav and one without, you need to make the approprate entry points. I.E. You need one with the side nave that pulls in the sidenav and then one that does not.

Also, I don't think our Drupal app modules have ever had page options. (They were not part of https://github.com/NCIOCPL/cgov-digital-platform/pull/1918, nor are they in the template now)